### ISSUE TITLE
Add the topic datagouvfr

### ISSUE THREAD

**Issue by pachevalier** [feddelegrand7/BARIS#1]
It's not easy to find packages on Github using only their name. I would recommend to add the topic datagouvfr to make it easier to find. 

Why did you name the package BARIS ? It's not very intuitive.

**Comment by feddelegrand7** [comment]
@pachevalier thank you very much for your feedback. 
- In fact, I've contacted 15 days ago datagouvfr through twitter in order to get some insights about how to display the package within their portal (I'm still waiting for their response) 
- I've submitted the package to CRAN and it's under review. I think when published, it will have more visibility. I've also made a quick tutorial about the package: https://ihaddadenfodil.com/post/working-with-the-french-open-data-portal-with-baris/
- For the name, yeah :D I'm really sorry about that I'm very bad at naming stuffs. I just thought about a movie in which an actor used to call Paris "BARIS" and I said why not haha.
